client/components/domains/registrant-extra-info/fr-form.jsx

```diff
@@ -120,7 +120,7 @@ class RegistrantExtraInfoForm extends React.PureComponent {
 
 	render() {
 		const translate = this.props.translate;
-		const registrantType = this.props.contactDetails.extra.registrantType;
+		const registrantType = get( this.props, 'contactDetails.extra.registrantType', 'individual' );
```

Comment:
I was doing this to get the form showing in the devdocs.

The problem I was trying to solve here was that I couldn't pass in appropriate props because the `connect` overwrote them. I was thinking over the weekend that a better way to do this is to export the onconnected form (by name) alongside the connected form (by default).